website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -14,6 +14,40 @@ You can create a Kubeflow Pipeline built entirely using SageMaker components, or
 
 There are two versions of SageMaker components - boto3 based v1 components and SageMaker Operator for K8s (ACK) based v2 components. 
```

Comment:
You can read more about the two versions in [SageMaker developer guide in AWS documentation](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html#kubeflow-pipeline-components).

----
line above 
The job parameters, status, logs, and outputs from SageMaker are accessible from the Kubeflow Pipelines UI. -> remove logs